tests/randomtest.c

```diff
@@ -480,7 +480,11 @@ exit(1);
     g_random_set_seed(now);
 
     FindFonts(dirs,exts);
+#ifdef _WIN32
+	mkdir(results_dir);
+#else
```

Comment:
Not sure I get this one. Why `mkdir` and not `GFileMkDir`? Does something provide that mapping?

Reply:
This is a completely standalone file that doesn't actually link to gutils, so I was reluctant to change that

Reply:
Well actually looking at this again, there's no way this would work on windows (or mingw) anyway, since it forks